tests/networks.spec.ts

```diff
@@ -4,148 +4,378 @@ import {
+    await page
+      .getByRole("button", {
+        name: /save/i,
+      })
+      .click();
+
+    await page.close();
+  });
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto("/ui/");
+    await page.getByRole("link", { name: "Networks", exact: true }).click();
+    await page.getByRole("button", { name: "Create network" }).click();
+    await page.getByRole("heading", { name: "Create a network" }).click();
+    await page.getByLabel("Type").selectOption("OVN");
+    await page.getByLabel("Name").click();
+    await page.getByLabel("Name").fill(network);
+    await page.getByLabel("Uplink").selectOption({ index: 2 });
+  });
+
+  test("configure main OVN network settings", async ({ page }) => {
+    const options = ["false", "true"];
+
+    await page
+      .getByPlaceholder("Enter description")
+      .fill(NETWORK_CONSTANTS.DESCRIPTION);
+
+    await page
+      .getByRole("row", {
+        name: "IPv4 address IPv4 address for",
+      })
+      .getByRole("button")
+      .click();
+    await page
+      .getByRole("row", {
+        name: "IPv6 address IPv6 address for",
+      })
+      .getByRole("button")
+      .click();
+
+    const radioButtons = await page.$$("input[type=radio]");
+    for (const button of radioButtons) {
+      await button.click({ force: true });
+    }
+
+    await page.getByLabel("IPv4 address").fill(NETWORK_CONSTANTS.IPV4_ADDRESS);
+    await page.getByLabel("IPv6 address").fill(NETWORK_CONSTANTS.IPV6_ADDRESS);
+
+    for (const option of options) {
+      await setOption(page, "IPv4 NAT", option);
+      await setOption(page, "IPv6 NAT", option);
+    }
+  });
+
+  test("set OVN bridge MTU and hardware address", async ({ page }) => {
+    await page
+      .getByRole("button", {
+        name: "Advanced",
+      })
+      .click();
+    await page
+      .getByLabel("Network form navigation")
+      .getByText("Bridge", { exact: true })
+      .click();
+
+    await setTextarea(page, "MTU", NETWORK_CONSTANTS.MTU);
+    await setTextarea(
+      page,
+      "Hardware address",
+      NETWORK_CONSTANTS.HARDWARE_ADDRESS,
+    );
+  });
+
+  test("configure OVN network DNS settings", async ({ page }) => {
+    await page
+      .getByRole("button", {
+        name: "Advanced",
+      })
+      .click();
+    await page
+      .getByLabel("Network form navigation")
+      .getByText("DNS", { exact: true })
+      .click();
+
+    await setTextarea(page, "DNS domain", NETWORK_CONSTANTS.DNS_DOMAIN);
+    await setTextarea(page, "DNS search", NETWORK_CONSTANTS.DNS_SEARCH);
+  });
+
+  test("set OVN network IPv4 DHCP and L3 options", async ({ page }) => {
+    const options = ["false", "true"];
+
+    await page
+      .getByRole("button", {
+        name: "Advanced",
+      })
+      .click();
+    await page
+      .getByLabel("Network form navigation")
+      .getByText("IPv4", { exact: true })
+      .click();
+
+    for (const option of options) {
```

Comment:
```suggestion

    const options = ["false", "true"];
    for (const option of options) {
```
Better define variables as close to their usage as possible to make this easier to read and maintain.